### DISCUSSION TITLE
images not showing, but they were earlier today

### DISCUSSION THREAD

**Discussion by peytons9431** [s0hv/foo_discord_rich#4] [Q&A]
i havent touched it and it just stopped working since around 10 hrs ago. i get this error in the console:

child "activity" fails because [child "assets" fails because [child "large_image" fails because ["large_image" length must be less than or equal to 256 characters long]]]

any ideas? thanks

**Comment by s0hv** [comment]
Sorry for taking so long to respond. Github doesn't seem to give me any notifications on new discussions. Anyways it seems like the url generated is too long. The 256 character character limit is set by discord so there is nothing that can be done by the plugin to fix this. If you're using imgur this should never be a problem as imgur urls are quite short. If you're using your own upload script for another service, you need to find a way to make the url shorter
